source/content/guides/drupal-9-migration/04-migrate-manual-d9.md

```diff
@@ -81,7 +81,7 @@ This doc uses the following aliases:
 1. List contrib modules and themes on your D8 site:
 
   ```bash{promptUser: user}
-  terminus drush $D8_SITE.dev -- pm:projectinfo --status=enabled --fields=name,version --format=table
+  terminus drush $D8_SITE.dev pm:projectinfo -- --status=enabled --fields=name,version --format=table
```

Comment:
These two commands should be the same.

Reply:
I read the original report again; perhaps the `--` in the first version was overlooked. It might be more obvious in the second position.